danger/action.yml

```
@@ -5,4 +5,7 @@ runs:
   using: composite
   steps:
     - shell: bash
+      env:
+        # We must specify the base URL for the GitHub API because the self-hosted runner overrides the default one
```

We can see in the code what we do. So I would remove the text. But to me the reason is not yet obvious. How the fact that "The self-hosted runner overrides the default one"  relates to what we do? It overrides with incorrect one and we override it again? It overrides unless we specify this one? 🤷 
```suggestion
        # The self-hosted runner overrides the default one, so ...
```

---

Yes, it overrides to incorrect one, and may use this env variable for another purpose.
I think, it was a cause why Danger added one more variable `DANGER_GITHUB_API_BASE_URL` and it has higher priority. I would rely on this env variable rather than common one: `GITHUB_URL`